validator-frontend/lang/en.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This is invalid json, the keys should likely be quoted. 

I would also suggest to prefix each message with where it's used (e.g. validation.checkingQR or something along those lines)